code/game/objects/items/toys.dm

```diff
@@ -904,8 +909,248 @@
 		'sound/weapons/cablecuff.ogg' = 1,
 	)
 
+/obj/item/toy/plushie/skrellplushie
+	name = "skrell plushie"
+	desc = "The latest from 'SoftSkrells.net', features its very own headpocket! Warble!"
+	icon_state = "plushie_skrell"
+	COOLDOWN_DECLARE(warble_cooldown)
+	var/obj/item/headpocket_item
+
+/obj/item/toy/plushie/skrellplushie/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>Alt-click to put something small inside the headpocket, or take an item out.</span>"
+
+/obj/item/toy/plushie/skrellplushie/AltClick(mob/user)
+	if(!Adjacent(user))
+		return
+	var/obj/item/I = user.get_active_hand()
+	if(I == src)
+		return
+	if(!I)
+		if(!headpocket_item)
+			return
+		to_chat(user, "<span class='notice'>You remove [headpocket_item] from [src].</span>")
+		headpocket_item.forceMove(get_turf(src))
+		user.put_in_hands(headpocket_item)
+		headpocket_item = null
+		return
+	if(I.w_class > WEIGHT_CLASS_SMALL)
+		to_chat(user, "<span class='warning'>You cannot fit [I] in [src]!</span>")
+		return
+	if(!iscarbon(user))
+		return
+	if(headpocket_item)
+		to_chat(user, "<span class='warning'>[src] already has an item in it's headpocket!</span>")
+		return
+	if(!user.drop_item())
+		to_chat(user, "<span class='warning'>You cannot slip [I] inside [src]!</span>")
+		return
+	user.visible_message("<span class='notice'>[user] places [I] into [src].</span>", "<span class='notice'>You place [I] into [src].</span>")
+	add_fingerprint(user)
+	I.forceMove(src)
+	headpocket_item = I
+
+/obj/item/toy/plushie/skrellplushie/Destroy()
+	QDEL_NULL(headpocket_item)
+	return ..()
+
+/obj/item/toy/plushie/skrellplushie/attack_self(mob/user)
+	if(!COOLDOWN_FINISHED(src, warble_cooldown))
+		return ..()
+	playsound(src,'sound/effects/warble.ogg', 10, 0)
+	visible_message("<span class='danger'>Warble!</span>")
+	COOLDOWN_START(src, warble_cooldown, 3 SECONDS)
+	return
+
+/obj/item/toy/plushie/humanplushie
+	name = "human plushie"
+	desc = "This plushie is slightly less popular than its counterparts. The designers obviously didn't find humans that endearing..."
+	icon_state = "plushie_human"
+	poof_sound = list('sound/weapons/thudswoosh.ogg' = 30,
+					'sound/goonstation/voice/male_scream.ogg' = 1)
+
+/obj/item/toy/plushie/borgplushie
+	name = "borg plushie"
+	desc = "The synthetic backbone of the station, rendered in plush form. Inbuilt flashlight included!"
+	icon_state = "plushie_borg"
+	var/on = FALSE
+
+/obj/item/toy/plushie/borgplushie/Initialize(mapload)
+	. = ..()
+	add_overlay(pick("plushie_borgjan", "plushie_borgsec", "plushie_borgmed", "plushie_borgmine", "plushie_borgserv", "plushie_borgassist", "plushie_borgengi"))
```

Comment:
dont do this in Initalize, use update_overlays instead

Reply:
Wouldn't this change their colour each time update_overlay is called?

Reply:
it can be set in initialize and used in update_overlays

Reply:
Done